sources/packages/web/src/components/common/IdleTimeChecker.vue

```diff
@@ -32,22 +46,35 @@ export default {
     },
   },
   setup(props: any) {
-    const lastActivityLogin = ref(new Date());
     const interval = ref();
     const extendTimeModal = ref({} as ModalDialog<boolean>);
     const { isAuthenticated } = useInstitutionAuth();
     const { getDatesDiff } = useFormatters();
+    const { executeLogout } = useAuth();
+    const countdown = ref(COUNT_DOWN_TIMER_FOR_LOGOUT);
+    const countdownInterval = ref(0);
+    const modalOpen = ref(false);
```

Comment:
is this name correct? is it the answer/rply from the modal?

Reply:
no, it's a variable to control/check whether the modal is open or not.